website/docs/r/sns_topic_policy.html.markdown

```diff
@@ -10,36 +10,59 @@ description: |-
 
 Provides an SNS topic policy resource
 
+~> **NOTE:** If a Principal is specified as just an AWS account ID rather than an ARN, AWS silently converts it to the ARN for the root user, causing future terraform plans to differ. To avoid this problem, just specify the full ARN, e.g. `arn:aws:iam::123456789012:root`
```

Comment:
It's good to call it out, but unfortunately this is a problem that appears in all kinds of policies of all AWS resources - so SNS Topic Policy isn't special in this case. 😑